src/Akka.Hosting.TestKit.Tests/HostingSpecSpec.cs

```diff
@@ -0,0 +1,64 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="HostingSpecSpec.cs" company="Akka.NET Project">
+//      Copyright (C) 2009-2022 Lightbend Inc. <http://www.lightbend.com>
+//      Copyright (C) 2013-2022 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using Akka.Actor;
+using Akka.Event;
+using Akka.TestKit.TestActors;
+using Xunit;
+using Xunit.Abstractions;
+using LogLevel = Microsoft.Extensions.Logging.LogLevel;
+
+namespace Akka.Hosting.TestKit.Tests
+{
+    public class HostingSpecSpec: TestKit
+    {
+        private enum Echo
+        { }
+
+        public HostingSpecSpec(ITestOutputHelper output)
+            : base(nameof(HostingSpecSpec), output, logLevel: LogLevel.Debug)
+        {
+        }
+
+        protected override Task ConfigureAkka(AkkaConfigurationBuilder builder, IServiceProvider provider)
```

Comment:
How would I populate the `IServiceProvider` in advance?

Reply:
Override the `ConfigureServices` method